public/locales/en/ui.json

```
@@ -794,7 +896,9 @@
   "shop.messages.purchaseSuccess": "{{itemName}} purchased!",
   "shop.messages.soundproofing": "Less noise, more peace. Harmony +5",
   "shop.messages.traitUnlocked": "Unlocked Trait: {{traitName}} ({{memberId}})",
+  "shop.messages.unknownItem": "Unknown Item",
```

Biome (2.4.13)

[error] The key shop.messages.unknownItem was already declared.

(lint/suspicious/noDuplicateObjectKeys)